web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -86,7 +90,7 @@ public class ModelBean {
     private static final long BATCH_WRITER_MAX_MEMORY = 10845760;
     private static final int BATCH_WRITER_MAX_THREADS = 2;
 
-    private static final HashSet<String> RESERVED_COLF_VALUES = Sets.newHashSet("e", "i", "ri", "f", "tf", "m", "desc", "edge", "t", "n", "h");
+    private static final HashSet<String> RESERVED_COLF_VALUES = Sets.newHashSet("e", "i", "ri", "f", "tf", "m", "desc", "edge", "t", "n", "h", "version");
```

Comment:
This fixes a preexisting bug: ```listModelNames``` method erroneously returns "version" (i.e., colf of the "edge_key" row) as an existing model